plexapi/client.py

```diff
@@ -540,20 +542,63 @@ def setStreams(self, audioStreamID=None, subtitleStreamID=None, videoStreamID=No
 
     # -------------------
     # Timeline Commands
-    def timeline(self, wait=1):
-        """ Poll the current timeline and return the XML response. """
-        return self.sendCommand('timeline/poll', wait=wait)
+    def timelines(self, wait=0):
+        """Poll the client's timelines, create, and return timeline objects."""
+        self.sendCommand(ClientTimeline.key, wait=1)
```

Comment:
Rather than using this workaround to "wakeup" client's not reporting timelines, I should just document Plex's buggy behavior for now. Still attempting to find a better way though.

Reply:
Imo the default argument should be 1 to increase the chance that we get a proper response timeline response. We don’t need it for work around as the point for the work around is just to make sure the command that we actually want to send is executed by pmp etc. Nice work btw. Thanks.

Reply:
Yeah, I think I'm happy with that for now. Will make the change. Thank you!

Reply:
Having the default be `wait=1` makes a timeline call potentially blocking. I don't think that's a safe default anymore.

Reply:
Just do what ever you prefer then. There are some info about the remote control api here: https://github.com/plexinc/plex-media-player/wiki/Remote-control-API (incase you havnt seen it)

Reply:
After further testing with this, the workaround and even adding a wait don't solve this issue. Initially it seemed those things helped, but it was a red herring. Was able to repeat the issue even with a wait and multiple workarounds.

It seems to be a bug on Plex's side and rather than add a workaround that causes timeline calls to block (sometimes for quite a bit), it's best to leave it as it was and just document the behavior for now. Will make the changes and document the issue.